File: app/back/src/main/java/mitei/mitei/investigate/report/balance/politician/dto/poli_org/balancesheet/keinen_henka/KeinenHenkaSurfaceAndSummaryByYearDto.java
package mitei.mitei.investigate.report.balance.politician.dto.poli_org.balancesheet.keinen_henka;

import java.io.Serializable;

import mitei.mitei.investigate.report.balance.politician.entity.OfferingBalancesheet01And20SurfaceEntity;
import mitei.mitei.investigate.report.balance.politician.entity.OfferingBalancesheet0702And0713SummaryEntity;

/**
 * 年単位の政治資金収支報告書の表紙Entityと集計Entity格納Dto
 */
public class KeinenHenkaSurfaceAndSummaryByYearDto implements Serializable { // NOPMD DataClass

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 収支報告書表紙Entity */
    private OfferingBalancesheet01And20SurfaceEntity offeringBalancesheet01And20SurfaceEntity;

    /** 収支報告書集計Entity */
    private OfferingBalancesheet0702And0713SummaryEntity offeringBalancesheet0702And0713SummaryEntity;

    /**
     * 収支報告書表紙Entityを取得する
     *
     * @return 収支報告書表紙Entity
     */
    public OfferingBalancesheet01And20SurfaceEntity getOfferingBalancesheet01And20SurfaceEntity() {
        return offeringBalancesheet01And20SurfaceEntity;
    }

    /**
     * 収支報告書表紙Entityを設定する
     *
     * @param offeringBalancesheet01And20SurfaceEntity 収支報告書表紙Entity
     */
    public void setOfferingBalancesheet01And20SurfaceEntity(
            final OfferingBalancesheet01And20SurfaceEntity offeringBalancesheet01And20SurfaceEntity) {
        this.offeringBalancesheet01And20SurfaceEntity = offeringBalancesheet01And20SurfaceEntity;
    }

    /**
     * 収支報告書集計Entityを取得する
     *
     * @return 収支報告書集計Entity
     */
    public OfferingBalancesheet0702And0713SummaryEntity getOfferingBalancesheet0702And0713SummaryEntity() {
        return offeringBalancesheet0702And0713SummaryEntity;
    }

    /**
     * 収支報告書集計Entityを設定する
     *
     * @param offeringBalancesheet0702And0713SummaryEntity 収支報告書集計Entity
     */
    public void setOfferingBalancesheet0702And0713SummaryEntity(
            final OfferingBalancesheet0702And0713SummaryEntity offeringBalancesheet0702And0713SummaryEntity) {
        this.offeringBalancesheet0702And0713SummaryEntity = offeringBalancesheet0702And0713SummaryEntity;
    }

}

File: app/front/src/dto/keinen_henka/keinenHenkaSurfaceAndSummaryByYearDto.ts
import OfferingBalancesheet01And20SurfaceInterface from "../../entity/offeringBalancesheet01And20SurfaceEntity";
import OfferingBalancesheet01And20SurfaceEntity from "../../entity/offeringBalancesheet01And20SurfaceEntity";
import OfferingBalancesheet0702And0713SummaryInterface from "../../entity/offeringBalancesheet0702And0713SummaryEntity";
import OfferingBalancesheet0702And0713SummaryEntity from "../../entity/offeringBalancesheet0702And0713SummaryEntity";

export default interface KeinenHenkaSurfaceAndSummaryByYearInterface {


}

export default class KeinenHenkaSurfaceAndSummaryByYearDto implements KeinenHenkaSurfaceAndSummaryByYearInterface {

    /** 収支報告書表紙Entity */
    offeringBalancesheet01And20SurfaceEntity: OfferingBalancesheet01And20SurfaceInterface;

    /** 収支報告書集計Entity */
    offeringBalancesheet0702And0713SummaryEntity: OfferingBalancesheet0702And0713SummaryInterface;

    /** 検索開始報告年 */
    houkokuNen: number;


    constructor() {

        this.offeringBalancesheet01And20SurfaceEntity = new OfferingBalancesheet01And20SurfaceEntity();
        this.offeringBalancesheet0702And0713SummaryEntity = new OfferingBalancesheet0702And0713SummaryEntity();
        this.houkokuNen = 0;
    }

}